test/convert.test.ts

```diff
@@ -81,7 +81,7 @@ describe("Convert", () => {
 
     it("parses URI without double slash in the beginning on Windows", () => {
       setProcessPlatform("win32")
-      expect(Convert.uriToPath("file:/x:/a/b/c/d.txt")).equals("x:\\a\\b\\c\\d.txt")
+      expect(Convert.uriToPath("file:/x:/a/b/c/d.txt").toLowerCase()).equals("x:\\a\\b\\c\\d.txt")
```

Comment:
Apparently the `URL` api uppercases drive letters on windows but I don't think that would actually be an issue since windows file paths are case insensitive.

Reply:
Does it work on other operating systems as expected?

Reply:
the issue is with the test. Other OSes keep the given case but windows uppercases the drive letters.

Really this test should only run on windows instead of changing the process to look like it is on windows.